langlib/lang.runtime/src/main/ballerina/runtime.bal

```diff
@@ -41,6 +41,16 @@ type CallStackElement record {|
 # The listener becomes a module listener of the module from which this
 # function is called.
 #
+# ```ballerina
+# var 'listener = object {
```

Comment:
We can use `runtime:DynamicListener` instead of `var`, right?

Reply:
Yes. But cannot we use `var` in this situation to make it simple?

Reply:
I don't see much of a difference. This will most likely be used with a variable of a specific listener type IMO, rather than var.

Reply:
ok, +1 to go with `runtime:DynamicListener`.